src/test/scala/nl.knaw.dans.easy.managedeposit/ReportGeneratorSpec.scala

```diff
@@ -209,8 +210,8 @@ class ReportGeneratorSpec extends TestSupportFixture
       s"${ deposit.storageSpace.toString }"
   }
 
-  private def createDeposit(depositorId: String, state: State, source: String): DepositInformation = {
-    DepositInformation(UUID.randomUUID().toString, "10.17026/dans-12345", Some(true), "FedoraId", depositorId, state, "", DateTime.now().minusDays(3).toString(), 2, 129000, "", source, "baggy")
+  private def createDeposit(depositorId: String, state: State, location: String): DepositInformation = {
+    DepositInformation(UUID.randomUUID().toString, "10.17026/dans-12345", Some(true), "FedoraId", depositorId, state, "", DateTime.now().minusDays(3).toString(), 2, 129000, "", location, "SWORD2", "baggy")
```

Comment:
Not sure, but I think you swapped `location` and `origin` here...

```suggestion
    DepositInformation(UUID.randomUUID().toString, "10.17026/dans-12345", Some(true), "FedoraId", depositorId, state, "", DateTime.now().minusDays(3).toString(), 2, 129000, "", "SWORD2", location, "baggy")
```